app/src/main/java/dev/patrickgold/florisboard/ime/text/keyboard/TextKey.kt

```diff
@@ -44,7 +45,8 @@ class TextKey(override val data: KeyData) : Key(data) {
         } else {
             computedData = computed
             computedPopups.clear()
-            computedPopups.hint = computedHint?.computeTextKeyData(evaluator)
+            computedPopups.hint = computedSymbolHint?.computeTextKeyData(evaluator)
+            computedPopups.merge(PopupSet(hint = computedNumericHint?.computeTextKeyData(evaluator)))
```

Comment:
```suggestion
            val symbolHint = computedSymbolHint
            val numericHint = computedNumericHint
            if (symbolHint != null && numericHint != null) {
                computedPopups.hint = symbolHint.computeTextKeyData(evaluator)
                numericHint.computeTextKeyData(evaluator)?.let { computedPopups.relevant.add(it) }
            } else if (symbolHint != null) {
                computedPopups.hint = symbolHint.computeTextKeyData(evaluator)
            } else if (numericHint != null) {
                computedPopups.hint = numericHint.computeTextKeyData(evaluator)
            }
```

Merging the hints this way avoids the need to temporarily allocate a new `PopupSet` just for merging in the numeric hint.